views/js/runner/plugins/tools/itemThemeSwitcher/itemThemeSwitcher.js

```diff
@@ -100,6 +111,31 @@ define([
             }
 
             /**
+             * register plugin's own shortcuts
+             */
+            function registerInnerShortcuts() {
+                var shortcuts = ['up', 'down', 'select'];
+                if (testConfig.allowShortcuts) {
+                    shortcuts.forEach(function (shortcutId) {
+                        shortcut.add(namespaceHelper.namespaceAll(pluginShortcuts[shortcutId], self.getName(), true), function () {
+                            testRunner.trigger('tool-themeswitcher-' + shortcutId);
+                        }, {
+                            avoidInput: true
+                        });
+                    });
+                }
+            }
+
+            /**
+             * unregister plugin's own shortcuts
+             */
+            function unregisterInnerShortcuts() {
+                shortcut.remove('up.' + self.getName());
+                shortcut.remove('down.' + self.getName());
+                shortcut.remove('select.' + self.getName());
```

Comment:
Not a change to do, but you could also use the same loop you did in  `registerInnerShortcuts()`.